client/src/generated/apis/AccountsApi.ts

```diff
@@ -162,7 +162,7 @@ export class AccountsApi extends runtime.BaseAPI implements AccountsApiInterface
             throw new runtime.RequiredError('principal','Required parameter requestParameters.principal was null or undefined when calling getAccountAssets.');
         }
 
-        const queryParameters: runtime.HTTPQuery = {};
+        const queryParameters: any = {};
```

Comment:
I don't understand why it is generating me this new type here when I haven't done any changes in those files.